portality/bll/services/application.py

```diff
@@ -251,6 +252,17 @@ def accept_application(self, application, account, manual_update=True, provenanc
 
         return j
 
+    def reject_update_request_of_journal(self, journal_id):
+        ur, jlock, alock = self.update_request_for_journal(journal_id)
+        if ur:
+            ur.add_note("Update request was automatically rejected because the associated journal was withdrawn or "
+                            "deleted.")
+            ur.save()
+            self.reject_application(ur, current_user._get_current_object())
+            return ur.id
```

Comment:
Since you have the update reqeust object, why not just return the whole thing?  This feels like the one use case you had for it required the id, so you just returned that, but the more general approach would be to return the object, then any caller can do what they want with it